Source/Core/InputCommon/GCAdapter.cpp

```diff
@@ -512,7 +528,10 @@ static bool CheckDeviceAccess(libusb_device* device)
                  desc.idVendor, desc.idProduct, 1);
 
   // In case of failure, capture the libusb error code into the adapter status
-  Common::ScopeGuard status_guard([&ret] { s_status = ret; });
+  Common::ScopeGuard status_guard([&ret] {
+    s_adapter_error = libusb_error(ret);
```

Comment:
I'd prefer for this to be a `static_cast` - it makes it more obvious that this isn't a function (like `libusb_strerror`), which is ambiguous since libusb uses the same naming scheme for enums and functions.